qna/DE-receipt-handout.md

```diff
@@ -0,0 +1,13 @@
+## Question
+
+What should i do when the order and invoice is processed including answer of the fiskaltrust.middleware but the posreceipt can't be print because of an internal technical reason of the POS?
```

Comment:
"What should I do when the order and receipt is processed including answer of the fiskaltrust.Middleware but the receipt can't be print because of an internal technical reason of the POS?"